bitwarden_license/bit-cli/src/admin-console/device-approval/device-approval.program.ts

```diff
@@ -3,13 +3,20 @@ import { program, Command } from "commander";
 import { BaseProgram } from "@bitwarden/cli/base-program";
 import { FeatureFlag } from "@bitwarden/common/enums/feature-flag.enum";
 
+import { ServiceContainer } from "../../service-container";
+
 import { ApproveAllCommand } from "./approve-all.command";
 import { ApproveCommand } from "./approve.command";
 import { DenyAllCommand } from "./deny-all.command";
 import { DenyCommand } from "./deny.command";
 import { ListCommand } from "./list.command";
 
+
 export class DeviceApprovalProgram extends BaseProgram {
+  constructor(serviceContainer: ServiceContainer) {
```

Comment:
You're passing in `ServiceContainer` as an argument, but not defining it as a property. Because the base class defines `this.ServiceContainer` using the OSS `ServiceContainer`, that's what the class is using throughout. You need to override the base class definition:

```suggestion
  constructor(protected serviceContainer: ServiceContainer) {
```

Reply:
Ah. That makes sense. Thank you!